test/Twig/Tests/Profiler/ProfileTest.php

```diff
@@ -69,9 +69,10 @@ public function testGetAddProfile()
     public function testGetDuration()
     {
         $profile = new Twig_Profiler_Profile();
+        usleep(1);
```

Comment:
Without this delay and running the test multiple times after each other on fast setup (and so with full op cache) this would fall over